internal/nginx/version.go

```diff
@@ -0,0 +1,100 @@
+package nginx
```

Comment:
seems like few lines here aren't covered by version_test.go, Is it possible to add few negatives cases?